### PR TITLE
Update collective.cover to 1.6b3

### PR DESCRIPTION
**Pull request by hvelarde**
closes https://github.com/plonegovbr/brasil.gov.portal/issues/382

### CODE REVIEW

File: buildout.d/versions.cfg [hunk 1]
```diff
@@ -117,8 +117,6 @@ pathtools = 0.1.2
 # BBB: https://github.com/plonegovbr/brasil.gov.tiles/commit/e00ed1db293278f66843c6cae8acb4ab9c2636d3
 # Essa versão é a que foi testada até hoje.
 plone.app.drafts = 1.0
-plone.app.imagetile = 1.0
```
Comment (idgserpro) [inline buildout.d/versions.cfg]: Por que esses dois estão sendo removidos? plone.app.imagetile e plone.app.texttile?

Comment (hvelarde) [reply buildout.d/versions.cfg]: porque ninguém depende deles.

**Comment by hvelarde** [comment]
@idgserpro podemos por favor mesclar isso aqui? aparentemente o Travis está falhando por algum problema de caching; preciso a ajuda de todos para reportar isso.

**Comment by idgserpro** [comment]
Achamos melhor o zc.buildout ser num outro PR (pois ele impacta todos os demais) https://github.com/plonegovbr/portal.buildout/pull/90. Favor aprovar e fazer um rebase aqui.

**Comment by rodfersou** [comment]
The initial content don't fill the variable `cover_layout`, what is expected in this new version of `collective.cover`.

Still looking how to fix it, but the fact is that this variable is created with `None` content now.

It looks like we will need to review the initial content https://github.com/plonegovbr/brasil.gov.portal/issues/220

**Comment by hvelarde** [comment]
@rodfersou esse é exatamente o problema que o release corrige: https://github.com/collective/collective.cover/issues/759

**Comment by rodfersou** [comment]
I doubt about it because the content are not being created by Plone form, it is a transmogrify pipeline that create the content.

**Pull request merged**